### ISSUE TITLE
InvalidArgumentError (see above for traceback): Received a label value of 19240 which is outside the valid range of [0, 128)

### ISSUE THREAD

**Issue by myinzack** [codedak/chatbot#9]
Hi,
am trying to train the model by executing the execute.py

The error is as follows:

InvalidArgumentError (see above for traceback): Received a label value of 17506 which is outside the valid range of [0, 128).  Label values: 17506 3 47 7 7 57 66 142 3888 78 7 316 3 612 84 169 9992 186 240 78 149 57 462 113 619 7 128 337 47 650 90 117 7 186 30 56 78 33 60 257 7 916 57 7 2363 753 3 5767 713 33 113 7 3 7 4 685 127 3 69 7 257 83 2063 20
	 [[node model_with_buckets/sequence_loss_1/sequence_loss_by_example/SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits (defined at /usr/local/lib/python3.6/site-packages/tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py:1100)  = SparseSoftmaxCrossEntropyWithLogits[T=DT_FLOAT, Tlabels=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"](model_with_buckets/embedding_attention_seq2seq_1/embedding_attention_decoder/attention_decoder/AttnOutputProjection/BiasAdd, model_with_buckets/sequence_loss_1/sequence_loss_by_example/Reshape)]]

I tried the following ways:
1.  add the seq2seq.py
    setattr(tf.contrib.rnn.GRUCell, '__deepcopy__', lambda self, _: self)
    setattr(tf.contrib.rnn.BasicLSTMCell, '__deepcopy__', lambda self, _: self)
    setattr(tf.contrib.rnn.MultiRNNCell, '__deepcopy__', lambda self, _: self)

2.  below change in seq2seq_model.py
def sampled_loss(labels, logits):
    labels = tf.reshape(labels, [-1, 1])
    return tf.nn.sampled_softmax_loss(tf.transpose(w), b, labels, logits, 
                                      num_samples,self.target_vocab_size) 

  
Still the issue is persisting pls suggest

FYI:
python 3.6
tensorflow 1.12.0
Thanks
vijay
